src/components/data/QueryAuth0.js

```diff
@@ -38,100 +40,117 @@ export default class QueryAuth0 extends React.PureComponent {
     userProfileReceived: PropTypes.func.isRequired,
   };
 
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
   async componentDidMount() {
-    const { authenticationFailed } = this.props;
+    this.checkForAuthentication();
+
+    window.addEventListener('focus', this.checkForAuthentication);
```

Comment:
This event listener should be cleaned up. I don't see that in `componentWillUnmount` though.